package.json

```diff
@@ -2,9 +2,39 @@
   "name": "reactjs-course-2",
   "version": "1.0.0",
   "description": "",
-  "main": "index.js",
+  "private": true,
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
You should configure test script in that task. And wright at leas one unit test e.g. for App.jsx

Reply:
Done!